submissions/adekemiAwomolo/dom/signup.js

```diff
@@ -0,0 +1,82 @@
+let submitForm = document.getElementById("formvalidation");
+    firstName = document.getElementById('firstname');
+    lastName = document.getElementById("lastname");
+    email = document.getElementById('email');
+    password = document.getElementById('password');
+    confirm_password = document.getElementById("password2");
+    mobile = document.getElementById('phone-no');
+    nameReg =/^[a-zA-Z]+$/i;
+    mobileReg =/^\(?\d{5}\)?-?\s*-?\d{6}$/;
+    emailReg=/^\w+([-]?\w+)*@\w+([-]?\w+)*(\w{2,10})+$/;
+    passwordReg =/(?=.*\d)(?=.*[a-z])(?=.*[a-zA-Z]).{6,}$/;
+    passWordMessage ='Password is invalid';
+
+    function required(field,RegExp, error, message, event){
+        if(field.value == ''){
+            field.nextElementSibling.innerHTML = "This field is required";
+            event.preventDefault();
+            return false;
+        }
+        else if(!field.value.match(RegExp)){
+            field.nextElementSibling.innerHTML = message;
+            event.preventDefault();
+            return false;
+        }
+        else{
+            field.nextElementSibling.innerHTML="";
+            return true;
+        }
+    
+}
+function emailValidation(event){
```

Comment:
This function name is misleading. This function is not validating email

Reply:
Good morning @adaugochi  i have corrected the function name, kindly review.Thank you